src/managers/GuildMemberManager.js

```diff
@@ -205,6 +205,24 @@ class GuildMemberManager extends CachedManager {
     return data.reduce((col, member) => col.set(member.user.id, this._add(member, cache)), new Collection());
   }
 
+  /**
+   * Options used for listing guild members.
+   * @typedef {Object} GuildListMembersOptions
+   * @property {Snowflake} after  Limit fetching members to those with an id greater than the supplied id
+   * @property {number} [limit=1] Maximum number of members to list
+   * @property {boolean} [cache=true] Whether or not to cache the fetched member(s)
+   */
+
+  /**
+   * Lists upto 1000 members of the guild.
+   * @param {GuildListMembersOptions} options Options for listing members
```

Comment:
Options are optional:
```suggestion
   * @param {GuildListMembersOptions} [options] Options for listing members
```